core/base/src/TColor.cxx

```diff
@@ -2243,6 +2264,44 @@ Int_t TColor::GetFreeColorIndex()
    return gHighestColorIndex+1;
 }
 
+////////////////////////////////////////////////////////////////////////////////
+/// List colors form ci1 to ci2
+
+void TColor::ListColors(Int_t ci1, Int_t ci2)
```

Comment:
What is the meaning of this function? Why would I want to list colors "from ci1 to ci2" as a user?

Reply:
typo: form --> from

Reply:
This function is useful for seeing the list of denied colors, what are their names which indices are free etc, etc ...
I had a macro doing that which I distributed many times to users asking for this functionality. I thought it was time to add it as a method. I will fix the typo.

Reply:
I don't understand. You say the function lists "denied colors", whereas the function doc states that it "lists colors from ci1 to ci2". I cannot understand the meaning of this sentence. Could you phrase it better?

Reply:
I did... see may last commit